File: Python/Program/Animal.py
class Animal:
    def __init__(self, name: str, diet: str, produce: str):
        self._name = name
        self._diet = diet
        self._produce = produce

    def get_name(self) -> str:
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_diet(self) -> str:
        return self._diet

    def set_diet(self, diet: str):
        self._diet = diet

    def get_produce(self) -> str:
        return self._produce

    def set_produce(self, produce: str):
        self._produce = produce

File: C++/Program/Animal.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Animal
{
private:
    string name;
    string diet;
    string produce;

public:
    // Constructor
    Animal() {}

    Animal(string name, string diet, string produce)
    {
        this->name = name;
        this->diet = diet;
        this->produce = produce;
    }

    // Getter and Setter for name
    string getName()
    {
        return name;
    }

    void setName(string name)
    {
        this->name = name;
    }

    // Getter and Setter for diet
    string getDiet()
    {
        return diet;
    }

    void setDiet(string diet)
    {
        this->diet = diet;
    }

    // Getter and Setter for produce
    string getProduce()
    {
        return produce;
    }

    void setProduce(string produce)
    {
        this->produce = produce;
    }
};